code/_onclick/hud/screen_objects.dm

```diff
@@ -260,7 +260,7 @@
 									if (!isnull(t.manipulated_by) && t.manipulated_by != C.real_name && findtext(t.desc,breathes))
 										contents.Add(t.air_contents.total_moles)	//Someone messed with the tank and put unknown gasses
 										continue					//in it, so we're going to believe the tank is what it says it is
-									if(t.air_contents.gas[breathes] && !t.air_contents.gas[MAT_PHORON])
+									if(t.air_contents.gas[breathes] && length(t.air_contents.gas) == 1)
```

Comment:
this going to cause internals button pick CO2 jetpacks / whatever really over air tanks (that are not pure oxygen) (yes those still exist)
maybe check that none of species set toxic gases are in there